ui/lib/core/addon/components/confirmation-modal.hbs

```diff
@@ -1,14 +1,6 @@
-<Modal
-  @title={{this.title}}
-  @onClose={{this.onClose}}
-  @isActive={{this.isActive}}
-  @type={{this.type}}
-  @showCloseButton={{true}}
->
+<Modal @title={{@title}} @onClose={{@onClose}} @isActive={{@isActive}} @type={{this.type}} @showCloseButton={{true}}>
```

Comment:
Is isActive being used anymore. looks like maybe you moved the mut action to the onClose action?

Reply:
I see it in the js file, but is it used on the actual hbs component at all?